components/HeaderBlock.vue

```diff
@@ -1,13 +1,20 @@
 <template>
-  <header class="container flex-row-reverse py-4 md:flex md:flex-row">
-    <div>
-      <client-only>
-        <RadioPlayer :show-album-art="true" :now-playing-uri="streamServer" />
-      </client-only>
+  <header class="flex-row-reverse items-center justify-between px-4 py-4 bg-white md:flex md:flex-row">
+    <div class="flex items-center">
+      <div class="block mr-4 cursor-pointer" @click="$router.push('/')">
+        <img src="/img/lahmacun-logo.png" alt="" class="w-24">
+      </div>
+      <div class="mr-8">
+        <client-only>
+          <RadioPlayer :show-album-art="true" :now-playing-uri="streamServer" />
```

Comment:
The `uri` part is really cool. Does this set-up mean we could - technically - easily use the same player for `streamServer` and then on click for `arcsi` and then back on streamServer. 

Maybe this is not the right place to discuss it, commenting on 1 line of code in the middle of PR but I'm thinking lot about the following: 

Let's say we have only 1 unified player. By default it plays the broadcast stream. User arrives and clicks on an arcsived episode to listen. This is fairly clear where the button is to listen to an old episode. The player switches from broadcast to playback. Here's my question: How/where can you switch back to listen to the radio? I've been thinking about this for some time and this is not clear to me.

Reply:
this is the way the current player works also. but any mp3 data stream should work with it yes. we could have some prop also eg. :arcsi-play="true" and handle all the differences between stream/arcsi with it
I was also thinking to have a unified player, we can discuss it on Saturday.